mm2src/mm2_main/src/lp_swap/swap_lock.rs

```diff
@@ -57,7 +56,14 @@ mod native_lock {
     #[async_trait]
     impl SwapLockOps for SwapLock {
         async fn lock(ctx: &MmArc, swap_uuid: Uuid, ttl_sec: f64) -> SwapLockResult<Option<SwapLock>> {
-            let lock_path = my_swaps_dir(ctx).join(format!("{}.lock", swap_uuid));
+            let lock_path = if cfg!(feature = "new-db-arch") {
+                ctx.global_dir().join("swap_locks").join(format!("{}.lock", swap_uuid))
+            } else {
+                ctx.global_dir()
```

Comment:
This was `my_swaps_dir` why change it to `global_dir`

Reply:
was just taking the easy route as this is non-critical data like cache and such. these locks don't have to reside in `my_swaps_dir` nor they are needed in backups. they can reside in the `global_dir` just fine and this avoids having to specifiy the `address` of `my_swaps_dir` (which we will have to do a DB query to get, or supply it from the caller if it's available)

Reply:
Makes sense. Thanks for the detailed explanation.